### PR TITLE
[Cards][Code addition][1437302]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Added json section for Carousel card(line nos.614-777 ).
Corrected syntax(line no.781)
Correction (line no.590)
Added line 606

### CODE REVIEW

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 1]
```diff
@@ -587,7 +587,7 @@ Bot Framework reference:
 
 Card collections are supported in Teams.
 
-Card collections are provided by the Bot Framework: `builder.AttachmentLayout.carousel` and `builder.AttachmentLayout.list`. These collections can contain Adaptive, Hero, or Thumbnail cards.
+Card collections: `builder.AttachmentLayout.carousel` and `builder.AttachmentLayout.list`. These collections can contain Adaptive, Hero, or Thumbnail cards.
```
Comment (KirtiPereira) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:590]: Confirm if the statement you have deleted is not required. If it is required to be mentioned then add it as a sentence to the text.
```Suggestion
Card collections: `builder.AttachmentLayout.carousel` and `builder.AttachmentLayout.list`. The Bot Framework provides the Card collections. These collections contain Adaptive, Hero, or Thumbnail cards.
```

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 2]
```diff
@@ -603,15 +603,182 @@ The [carousel layout](/azure/bot-service/dotnet/bot-builder-dotnet-add-rich-card
 > [!NOTE]
 > A carousel can display a maximum of 10 cards per message.
 
+### Properties of a Carousel card
+
+Properties are the same as for the hero or thumbnail card.
```
Comment (KirtiPereira) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:608]:
```Suggestion
Properties of a Carousel card are same as those of the Hero and Thumbnail cards.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md (hunk 1 above)
Comment (Rajeshwari-v) [reply msteams-platform/task-modules-and-cards/cards/cards-reference.md:590]: Confirmed with SME that  " The Bot Framework provides the Card collections." is not required. So, removing this line. Accepted rest of the suggestion.

**Review by Nikitha-MSFT — DISMISSED** [review]
Reviewed and Approved.

**Review by KirtiPereira — DISMISSED** [review]
Approved for merging.

**Review by KirtiPereira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Changes seem fine. Approving the file.

**Pull request merged**